requirements.txt

```diff
@@ -1,10 +1,11 @@
 hydra-core>=1.2.0
 wandb>=0.13
-tree-sitter
-pygraphviz
-graphviz
 numpy
 transformers>=4.22.0
 tokenizers
+torch
+swig
```

Comment:
What is this for?

Reply:
Without this `swig` (I don't know what it is), there will be an error from box2d
`error: command 'swig' failed: No such file or directory`.
Same goes for `box2d-py` which seems to be needed to avoid an error in sodaracer env about an environment variable or something.

There are a lot of strange stuff around this box2d thing.